src/Atmos/Model/ref_state.jl

```diff
@@ -131,10 +111,8 @@ reaches the minimum specified temperature `T_min`
 $(DocStringExtensions.FIELDS)
```

Comment:
Change documentation above: Replace "A temperature profile that has uniform potential temperature θ until it
reaches the minimum specified temperature `T_min`"

by

A temperature profile that has uniform dry potential temperature θ.